src/Symfony/Component/VarDumper/Server/DumpServer.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Server;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\VarDumper\Cloner\Data;
+
+/**
+ * A server collecting Data clones sent by a ServerDumper.
+ *
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class DumpServer
+{
+    const HOST_ENV_VAR = 'VAR_DUMPER_SERVER';
```

Comment:
Yes, we could leverage Symfony's env support for this, hence no need for the previous commit. But the rational here to me is that you probably don't need to make the env var itself configurable and when working with multiple projects, you don't have to ensure the same env var is used. Just set it and it works out of the box.

Still, that'll be the most flexible and would avoid hardcoding this behavior, while we could just configure in the standard-edition and flex recipe:

```yml
parameters:
    env(VAR_DUMPER_SERVER): 'tcp://0.0.0.0:9912'

debug:
    server_dump:
        host: '%env(VAR_DUMPER_SERVER)%'
```

@fabpot : What do you think?